neurom/analysis/morphtree.py

```diff
@@ -44,8 +44,13 @@
 
 def path_length(tree):
```

Comment:
I tried a recursive solution, which could greatly benefit from memoization, but the maximum recursion depth was reached. Maybe this thing can be expressed recursively in terms of sections instead of segments.